x-pack/plugins/profiling/public/components/flamegraph.tsx

```diff
@@ -112,60 +113,62 @@ function FlameGraphTooltip({
   comparisonTotalSamples?: number;
 }) {
   return (
-    <EuiPanel>
-      <EuiFlexGroup
-        direction="column"
-        gutterSize="m"
-        style={{
-          overflowWrap: 'anywhere',
-        }}
-      >
-        <EuiFlexItem>{label}</EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFlexGroup direction="column" gutterSize="xs">
-            {isRoot === false && (
-              <>
-                <TooltipRow
-                  label={i18n.translate('xpack.profiling.flameGraphTooltip.inclusiveCpuLabel', {
-                    defaultMessage: `CPU incl. subfunctions`,
-                  })}
-                  value={countInclusive / totalSamples}
-                  comparison={
-                    isNumber(comparisonCountInclusive) && isNumber(comparisonTotalSamples)
-                      ? comparisonCountInclusive / comparisonTotalSamples
-                      : undefined
-                  }
-                  formatAsPercentage
-                  showChange
-                />
-                <TooltipRow
-                  label={i18n.translate('xpack.profiling.flameGraphTooltip.exclusiveCpuLabel', {
-                    defaultMessage: `CPU`,
-                  })}
-                  value={countExclusive / totalSamples}
-                  comparison={
-                    isNumber(comparisonCountExclusive) && isNumber(comparisonTotalSamples)
-                      ? comparisonCountExclusive / comparisonTotalSamples
-                      : undefined
-                  }
-                  formatAsPercentage
-                  showChange
-                />
-              </>
-            )}
-            <TooltipRow
-              label={i18n.translate('xpack.profiling.flameGraphTooltip.samplesLabel', {
-                defaultMessage: `Samples`,
-              })}
-              value={countInclusive}
-              comparison={comparisonCountInclusive}
-              formatAsPercentage={false}
-              showChange
-            />
-          </EuiFlexGroup>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiPanel>
+    <TooltipContainer>
```

Comment:
Same as before: are using the container here to keep the rounder border?

Reply:
Let's see what O11y folks think about that

Reply:
See comments from `stacked_bar_chart.tsx` above.